internal/doculint/doculint.go

```diff
@@ -387,8 +387,9 @@ func validateFuncDecl(reporter nolint.Reporter, expr *ast.FuncDecl) {
 		return
 	}
 
-	if !strings.HasPrefix(strings.TrimSpace(expr.Doc.Text()), expr.Name.Name) {
-		reporter.Reportf(expr.Pos(), "comment for function \"%s\" should begin with \"%s\"", expr.Name.Name, expr.Name.Name)
+	// Enforce a space after the function name.
+	if !strings.HasPrefix(strings.TrimSpace(expr.Doc.Text()), expr.Name.Name+" ") {
```

Comment:
Should we also enforce a space before?

Reply:
Won't gofmt complain on that?

Reply:
Yeah I am 99% sure that 1.9 made that a `gofmt` setting. That's what pushed us to change stencil comment format.

Reply:
Fair point :P